inst/tinytest/test_gprop.R

```diff
@@ -8,6 +12,10 @@ expect_true(nrow(res$interest_by_city) > 0)
 expect_true(nrow(res$related_topics) > 0)
 expect_true(nrow(res$related_queries) > 0)
 
+if(!curl::has_internet()) {
+  exit_file("Skipping tests for lack of internet")
```

Comment:
You only need them once per file.  When you hit `exit_file()` the file is in fact exited.